eng/scripts/Language-Settings.ps1

```diff
@@ -5,6 +5,7 @@ $PackageRepository = "NPM"
 $packagePattern = "*.tgz"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/js-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=javascript%2F&delimiter=%2F"
+$Track1SdkTypeExclusions = @("@azure/event-processor-host")
```

Comment:
What is this needs?

Reply:
event-processor-host has sdk-type set to ```client``` but this is not new SDK, I have discussed this with @ramya-rao-a earlier and she suggested to have an exclusion for now instead of changing sdk-type property value to empty. We check  for this property presence to identify whether it new or old SDK type.

Reply:
@ramya-rao-a I'd prefer to not have these type of special cases in random places. Is there a reason we cannot set sdk-type to data or something other then client for this library? That is why we have that property in the project.